apps/scan/backend/src/printing/readiness_report.ts

```diff
@@ -34,6 +34,7 @@ export async function saveReadinessReport({
   const { store } = workspace;
   const generatedAtTime = new Date(getCurrentTime());
   const electionRecord = store.getElectionRecord();
+  const markThresholds = store.getSystemSettings()?.markThresholds ?? undefined;
```

Comment:
Since `store.getSystemSettings()` [returns `SystemSettings | undefined`](https://github.com/votingworks/vxsuite/blob/65e2910634bc3cab072f4f40cd986432b4dca88c/apps/scan/backend/src/store.ts#L919), `store.getSystemSettings()?.markThresholds` will always evaluate to a truthy value or `undefined`. The nullish coalescing operator `??` isn't necessary in this case.

Reply:
Ah, true. Will update accordingly